apps/backend/src/libs/modules/config/base-config.module.ts

```diff
@@ -76,6 +76,14 @@ class BaseConfig implements Config {
 					format: Number,
 				},
 			},
+			ENCRYPTION: {
+				SALT_ROUNDS: {
+					default: 10,
```

Comment:
```suggestion
					default: null,
```
let's avoid default values